### DISCUSSION TITLE
Write to file

### DISCUSSION THREAD

**Discussion by therealmrnerd** [EDCD/EDDI#2479] [EDDI Standalone Q&A]
Hi there,

Thanks for the great work. I was wondering if it was possible to use a script to write an event to a file in simple data format eg if docked write to an ini file with docked : true, or docked : false;  if in gear is down write to an ini file with landinggear : true, or landinggear : false; and update these values. As an ini it's very easily read by a few bits of software including Sammi/Lioranboard Board, a streamdeck, adding in full OBS stream support to Elite Dangerous as well as allowing it to function as in game controller.

Within your scripts I only find support for TTS so not sure if anything can be added via a script or if it needs to be done as an additional API output.

**Comment by Tkael** [comment]
I'm afraid we do not support writing arbitrary data to arbitrary files at this time. If you use VoiceAttack then you could monitor events / variables via VoiceAttack and write to files from there.
